github/ci/services/grafana/manifests/grafana.yaml

```diff
@@ -3574,7 +3593,7 @@ data:
       "timezone": "",
       "title": "e2e jobs overview V2",
       "uid": "efpTS3t4z",
-      "version": 8,
+      "version": 1,
```

Comment:
Any reason for the version change or was this just how it exported?

Reply:
The latter - I don't have any idea why the export changed the version.

Reply:
If things break (which I doubt) we can just revert though.